sdk/eventhub/Azure.Messaging.EventHubs/src/Compatibility/TrackOneEventHubConsumer.cs

```diff
@@ -8,7 +8,6 @@
 using System.Threading.Tasks;
 using Azure.Core;
 using Azure.Messaging.EventHubs.Core;
-using Azure.Messaging.EventHubs.Metadata;
 
 namespace Azure.Messaging.EventHubs.Compatibility
```

Comment:
Do we still need Track One code?

Reply:
For the moment.   I want to let the new stuff bake a bit.   I'm likely to remove it as part of the client hierarchy work, assuming nightly runs still look tight.